container/src/LuigiContainer.svelte

```diff
@@ -5,6 +5,7 @@
   export let context;
   export let label;
   export let webcomponent;
+  export let locale;
```

Comment:
maybe also currentTheme

Reply:
applyCoreStateData(data) {
 
      activeFeatureToggleList: LuigiFeatureToggles.getActiveFeatureToggleList(),
      currentLocale: LuigiI18N.getCurrentLocale(),
      currentTheme: LuigiTheming.getCurrentTheme()
      
      can also be good candidates